### PR TITLE
chore: Generate dashboard configmap from separate json files

### LINKED ISSUE #420: Autogenerate grafana_dashboard_dashboards_noparse.yaml
Currently it's somewhat of a pain to add modifications to dashboards because these are JSON inside yaml files, formatting issues etc etc.

It'll be nice if we can have some `dashboard` directory somewhere where we store dashboards in json. and we can run some script to auto generate grafana_dashboard_dashboards_noparse.yaml from them. then we can apply it as we're currently applying it.

related #364 

https://github.com/prometheus/test-infra/blob/master/prombench/manifests/cluster-infra/grafana_dashboard_dashboards_noparse.yaml

### PR DESCRIPTION
**Pull request by ArthurSens**
While doing the [release for v2.52](https://github.com/prometheus/prometheus/pull/13969), I wanted to extend the prombench dashboard with more information that was needed to evaluate the release.

In the process of doing the contribution here, I realized how painful it is to contribute code to a json nested inside a configmap yaml. 

I want to make this process easier first, and contribute the new dashboard panel after :)

Fix #420

### CODE REVIEW

File: prombench/manifests/cluster-infra/grafana_dashboard_dashboards_noparse.yaml [hunk 1]
```diff
@@ -1573,36 +1573,36 @@ data:
               "steppedLine": false,
               "targets": [
               {
-                  "expr": "node_memory_MemTotal_bytes{job=\"node-exporter\",namespace=\"prombench-[[pr-number]]\",node=~\"(test+).*\"}",
+                  "expr": "node_memory_MemTotal_bytes{job=\"node-exporter\",namespace=\"prombench-${prNumber}\",node=~\"(test+).*\"}",
```
Comment (ArthurSens) [inline prombench/manifests/cluster-infra/grafana_dashboard_dashboards_noparse.yaml]: I don't understand why we're seeing this diff 🤨 

In master I see this line with `${prNumber}`

https://github.com/prometheus/test-infra/blob/475461d4fe2cd957af054c63e36b77d97eec4a67/prombench/manifests/cluster-infra/grafana_dashboard_dashboards_noparse.yaml#L1576

Same thing for all other changes in this file

**Comment by ArthurSens** [comment]
Anyone up for a review? 😬

File: prombench/manifests/cluster-infra/grafana_dashboard_dashboards_noparse.yaml (hunk 1 above)
Comment (geekodour) [reply prombench/manifests/cluster-infra/grafana_dashboard_dashboards_noparse.yaml]: change was introduced here: https://github.com/prometheus/test-infra/commit/506588a5d256f983cb2060dd6cdd1e3b76456f75 maybe your local master was cloned before this one got in?

Think a local rebase to latest master should fix it?

Comment (ArthurSens) [reply prombench/manifests/cluster-infra/grafana_dashboard_dashboards_noparse.yaml]: That was it, thanks :)

**Review by bwplotka — APPROVED** [review]
Great job, thanks!

**Pull request merged**